### PR TITLE
Convenience Makefile credential rules for monitoring cluster

### PR DESCRIPTION
**Pull request by adrcunha**

### CODE REVIEW

File: tools/monitoring/Makefile [hunk 1]
```diff
@@ -12,5 +12,16 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+CLUSTER       ?= monitoring
```
Comment (chaodaiG) [inline tools/monitoring/Makefile]: should this be `CLUSTER = monitoring` to avoid accident?

Comment (adrcunha) [reply tools/monitoring/Makefile]: 1. This is intended to give flexibility.
1. I don't think "make CLUSTER=prow" would configure an accident. :)

Comment (chaodaiG) [reply tools/monitoring/Makefile]: gotcha

**Pull request merged**